src/components/molecules/ZopaFooter/ZopaFooter.tsx

```diff
@@ -48,7 +48,7 @@ const ZopaFooter = ({
         {theme.footer.showFooterLinks && (
           <FlexRow className="mb-6">
             <FlexCol xs={12} s={6} l={3} className="mb-8 s:mb-9">
-              <Heading className="mb-4 s:mb-6">What we do</Heading>
+              <Heading className="mb-4 s:mb-6 mt-0">What we do</Heading>
```

Comment:
I think adding the `as="h4"` property makes more sense being added here than in its own component makes it clearer and more readable wdyt?

Also I am not sure why we have `text: uppercase` on them might be worth chatting to design about that since we don't really do it anywhere else to my knowledge?

Reply:
I'll check with design about this but leave the uppercase for now.
I think I was trying to avoid duplication but having as="h4" makes a lot more sense